### PR TITLE
MSEARCH-42: [PT2] Remove previous instance-only Kafka listener. 

### PR DESCRIPTION
**Pull request by bohdan-suprun**
* Remove the old instances Kafka listener.
* Adjust tests.

### CODE REVIEW

File: src/test/java/org/folio/search/integration/KafkaMessageListenerTest.java [hunk 1]
```diff
@@ -66,4 +63,37 @@ void handleEvents() {
       new ConsumerRecord<>("inventory.holding-record", 0, 0, null, holdingBody2)
     ));
   }
+
+  @ValueSource(strings = {"CREATE", "UPDATE", "REINDEX"})
+  @ParameterizedTest
+  void handleEvents_onlyOfKnownTypes(String eventType) {
```
Comment (hjiebsco) [inline src/test/java/org/folio/search/integration/KafkaMessageListenerTest.java:69]: Do not see `_` in method name often.

File: src/test/java/org/folio/search/integration/KafkaMessageListenerTest.java [hunk 2]
```diff
@@ -66,4 +63,37 @@ void handleEvents() {
...
+
+    messageListener.handleEvents(List.of(
+      new ConsumerRecord<>(INVENTORY_INSTANCE_TOPIC, 0, 0,
+        UUID.randomUUID().toString(), resourceBody)));
+
+    verify(indexService).indexResources(any());
+  }
+
+  @Test
+  void handleEvents_instanceIdIsPartitionKeyForReindex() {
```
Comment (hjiebsco) [inline src/test/java/org/folio/search/integration/KafkaMessageListenerTest.java:83]: Do not see `_` in method name often.

**Review by hjiebsco — APPROVED** [review]

File: src/test/java/org/folio/search/integration/KafkaMessageListenerTest.java (hunk 1 above)
Comment (bohdan-suprun) [reply src/test/java/org/folio/search/integration/KafkaMessageListenerTest.java]: Renamed.

File: src/test/java/org/folio/search/integration/KafkaMessageListenerTest.java (hunk 2 above)
Comment (bohdan-suprun) [reply src/test/java/org/folio/search/integration/KafkaMessageListenerTest.java]: Renamed as well.

**Review by pfilippovEPAM — APPROVED** [review]

**Pull request merged**